src/components/grid-aware/Hero/Shape.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="20" viewBox="0 0 16 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you rename this button to something like `play-button.svg`. `Shape.svg` is not very descriptive.

Reply:
got it